digit-health-services/health-sync-service/src/main/java/org/digit/health/sync/web/models/CampaignData.java

```diff
@@ -0,0 +1,18 @@
+package org.digit.health.sync.web.models;
+
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+@JsonSubTypes({ @JsonSubTypes.Type(value = HouseholdRegistration.class,
+        name = "householdRegistration"),
+        @JsonSubTypes.Type(value = Delivery.class, name = "delivery")})
```

Comment:
We should also have Registration and IndividualRegistration as possible types?

Reply:
We will add these when we implement the actual Registration service. For the first use case, the scope is to only log the request in Registration service.